crates/nu-parser/src/errors.rs

```diff
@@ -4,10 +4,10 @@ use std::fmt::Debug;
 #[derive(Debug)]
 pub struct ParseError<T: Debug> {
     /// An informative cause for this parse error
-    pub(crate) cause: nu_errors::ParseError,
+    pub cause: nu_errors::ParseError,
 
     /// What has been successfully parsed, if anything
-    pub(crate) partial: Option<T>,
```

Comment:
If we want to keep these public to only the crate, perhaps a `From` implementation could work. Not sure if there's a preference.